utils/firestore.js

```diff
@@ -419,44 +419,78 @@ const removeParticipantsDataDestruction = async () => {
-        const destroyDataCId = fieldMapping.participantMap.destroyData.toString();
-        const dateRequestedDataDestroyCId = fieldMapping.participantMap.dateRequestedDataDestroy.toString();
-        const destroyDataCategoricalCId = fieldMapping.participantMap.destroyDataCategorical.toString();
-        const requestedAndSignCId = fieldMapping.participantMap.requestedAndSign;
+        const stubFieldArray = [ "query", "pin", "token", "state", "Connect_ID", "471168198", "736251808", "436680969", "480305327", "564964481", "795827569", "544150384", "371067537", "454205108", "454445267", "919254129", "412000022", "558435199", "262613359", "821247024", "914594314", "747006172", "659990606", "299274441", "919699172", "141450621", "576083042", "431428747", "121430614", "523768810", "639172801", "175732191", "150818546", "624030581", "285488731", "596510649", "866089092", "990579614", "131458944", "372303208", "777719027", "620696506", "352891568", "958588520", "875010152", "404289911", "637147033", "734828170", "715390138", "538619788", "153713899", "613641698", "407743866", "831041022", "269050420", "359404406", "119449326", "304438543", "912301837", "130371375", "765336427", "479278368", "826240317", "693626233", "104278817", "744604255", "268665918", "592227431", "399159511", "231676651", "996038075", "506826178", "524352591", "902332801", "457532784", "773707518", "577794331", "883668444", "827220437", "699625233", ];
+        const subStubFieldArray = ["firstName", "lastName", "studyId", "uid"];
+        const dataHasBeenDestroyed =
+            fieldMapping.participantMap.dataHasBeenDestroyed.toString();
+        const destroyDataCId =
+            fieldMapping.participantMap.destroyData.toString();
+        const dateRequestedDataDestroyCId =
+            fieldMapping.participantMap.dateRequestedDataDestroy.toString();
+        const destroyDataCategoricalCId =
+            fieldMapping.participantMap.destroyDataCategorical.toString();
+        const requestedAndSignCId =
+            fieldMapping.participantMap.requestedAndSign;
```

Comment:
Suggestion: Add few comments to explain the purpose and logic of certain sections of the code. This will make it easier for other developers (and your future self) to understand the code.